contracts/HydraStaking/HydraStaking.sol

```diff
@@ -276,13 +291,13 @@ contract HydraStaking is
      */
     function _distributeReward(
         uint256 epochId,
-        Uptime calldata uptime,
+        Uptime memory uptime,
         uint256 fullRewardIndex,
         uint256 totalSupply,
         uint256 totalBlocks
     ) private returns (uint256 reward) {
         if (uptime.signedBlocks > totalBlocks) {
-            revert DistributeRewardFailed("SIGNED_BLOCKS_EXCEEDS_TOTAL");
+            uptime.signedBlocks = totalBlocks;
```

Comment:
@SamBorisov Theoretically, we must not have such a case, but we might have a bad actor that changes the code and gives more signed blocks for his own validator in order to get more rewards. However, if we keep the revert, it will just drop, but I suppose @R-Santev changed it to just get totalBlocks to avoid errors when the node is distributing rewards.